nodes/vr_xrv9k/vr-xrv9k.go

```diff
@@ -59,7 +59,7 @@ func (n *vrXRV9K) Init(cfg *types.NodeConfig, opts ...nodes.NodeOption) error {
 		"USERNAME":           defaultCredentials.GetUsername(),
 		"PASSWORD":           defaultCredentials.GetPassword(),
 		"CONNECTION_MODE":    nodes.VrDefConnMode,
-		"VCPU":               "2",
+		"VCPU":               "4",
 		"RAM":                "16384",
```

Comment:
As simple as removing these two lines I guess?

Reply:
Yes, but first ensuring that the vrnetlab has these defaults set

Reply:
Yep gotcha 👍